lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_backmerge_pull_request_action.rb

```diff
@@ -0,0 +1,168 @@
+require 'fastlane/action'
+require_relative '../../helper/github_helper'
+
+module Fastlane
+  module Actions
+    class CreateReleaseBackmergePullRequestAction < Action
+      DEFAULT_BRANCH = 'trunk'.freeze
+
+      def self.run(params)
+        token = params[:github_token]
+        repository = params[:repository]
+        source_branch = params[:source_branch]
+        default_branch = params[:default_branch]
+        target_branches = params[:target_branches]
+        labels = params[:labels]
+        milestone_title = params[:milestone_title]
+        reviewers = params[:reviewers]
+        team_reviewers = params[:team_reviewers]
+
+        if target_branches.include?(source_branch)
+          UI.user_error!('`target_branches` must not contain `source_branch`')
+        end
+
+        github_helper = Fastlane::Helper::GithubHelper.new(github_token: params[:github_token])
+        target_milestone = milestone_title.nil? ? nil : github_helper.get_milestone(repository, milestone_title)
+
+        final_target_branches = if target_branches.empty?
+                                  determine_target_branches(release_version: source_branch.delete('release/'), default_branch: default_branch)
+                                else
+                                  target_branches
+                                end
+
+        final_target_branches.map do |target_branch|
+          Fastlane::Helper::GitHelper.checkout_and_pull(source_branch)
+
+          create_backmerge_pr(
+            token: token,
+            repository: repository,
+            title: "Merge #{source_branch} into #{target_branch}",
+            head_branch: source_branch,
+            base_branch: target_branch,
+            labels: labels,
+            milestone: target_milestone&.number,
+            reviewers: reviewers,
+            team_reviewers: team_reviewers
+          )
+        end
+      end
+
+      def self.determine_target_branches(release_version:, default_branch:)
```

Comment:
How simple and clearly-named this helper function might seem, I still think it'd be nice to add some YARD to it, especially to document the parameters and their expected types.

Same goes for `self.create_backmerge_pr`

Reply:
Done on 5a1d57a34966ff6d18e4e5566863af1f955d3a32